.github/workflows/build.yml

```diff
@@ -3,24 +3,28 @@ on:
   pull_request:
   push:
     branches: [ master ]
+
+env:
+  FOUNDRY_PROFILE: ci 
```

Comment:
According to https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#env it sets the `FOUNDRY_PROFILE` for all jobs making the PR runs as long as the `master` runs. This is unfortunate because work on the PRs is often slowed down by the CI. This is why there were 2 different configurations: a quick one for the PRs and a long one for `master` after the PR is merged and nobody waits for the CI to finish anymore. If we set that variable in https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsenv, we can restore that setup.

Reply:
They must've added it like today but fuzz runs can now be passed using the env variable: https://onbjerg.github.io/foundry-book/reference/config.html#fuzz_runs If it overrides `foundry.toml` (it should), then we don't need a separate profile.

Reply:
I might just have overseen that as I was only searching for a flag that I can pass to the command.